charts/ceph-csi-rbd/templates/provisioner-psp.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.provisioner.podSecurityPolicy.enabled -}}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ include "ceph-csi-rbd.provisioner.fullname" . }}
+  labels:
+    app: {{ include "ceph-csi-rbd.name" . }}
+    chart: {{ include "ceph-csi-rbd.chart" . }}
+    component: {{ .Values.provisioner.name }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  allowPrivilegeEscalation: true
+  allowedCapabilities:
+    - 'SYS_ADMIN'
+  fsGroup:
+    rule: RunAsAny
+  privileged: true
+  runAsUser:
+    rule: RunAsAny
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  volumes:
+    - 'configMap'
+    - 'emptyDir'
+    - 'projected'
+    - 'secret'
+    - 'downwardAPI'
+    - 'hostPath'
```

Comment:
please check `hostPath` and downwardAPI is required for provisioner for both cephfs and rbd?

Reply:
`hostPath` is required. `projected` and `downwardAPI` are not required. However they are in the list of recommended minimum set of volumes for PSPs in Kubernetes documentation. https://kubernetes.io/docs/concepts/policy/pod-security-policy/#volumes-and-file-systems.

I can still remove them if that's what you prefer though.

Reply:
let's keep it, Thanks